cli/download/download.go

```diff
@@ -107,7 +107,7 @@ func downloadFile(uri string) error {
 	}
 
 	ctx := context.Background()
-	if apiKey, err := storage.ReadRepoConfig("api-key"); err == nil {
+	if apiKey, _ := storage.ReadRepoConfig("api-key"); apiKey != "" {
```

Comment:
Seems like it'd be more correct to check err != nil && str != ""

Reply:
done